site/content/en/concepts/api-conventions/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "API Conventions"
+linkTitle: "API Conventions"
+weight: 2
+type: docs
+description: >
+   Kpt API Conventions
+---
+
+## Resource Annotations
```

Comment:
I think this is fine.  Docs duplication can be ok when -- contextualizing docs, increasing visibility, publishing to another audience.  E.g. blogposts usually duplicate content.

Reply:
It could become a PITA if there's churn, worst case scenario is they diverge from each other.

Reply:
I'm not worried about them becoming inconsistent -- the annotations are part of the API.  It could miss annotations added to kustomize, or kpt could annotations that kustomize doesn't have..

Reply:
How about `config.kubernetes.io/local-config`? it's upstream but not here.

Reply:
the kpt live commands don't respect it yet.